docs/examples/hypervisors/kvm/README.md

```diff
@@ -0,0 +1,109 @@
+KVM Virtualization
+==================
+
+This sample uses Terraform to demonstrate the Bring Your Own Hypervisor model to deploy custom KVM based Virtual Machines on top of a KVM hypervisor on Oracle Cloud Infrastructure.
+
+In this example we demonstrate the automatic installation of a [Web Application Firewall - WAF](https://en.wikipedia.org/wiki/Web_application_firewall) running on Oracle Cloud Infrastructure (OCI) based on the network topology described on the diagram below, but it can be easily adapted to automate the installation of any Virtual Machine based on KVM:
```

Comment:
In this example we demonstrate the automatic installation of a guest VM on a Bare Metal instance in Oracle Cloud Infrastructure (OCI). The guest VM will be configured with multiple [VNICs](https://docs.us-phoenix-1.oraclecloud.com/Content/Network/Tasks/managingVNICs.htm) to be setup a [Web Application Firewall - WAF](https://en.wikipedia.org/wiki/Web_application_firewall) or a [NAT gateway](https://en.wikipedia.org/wiki/Network_address_translation) with the network topology as described in the diagram below. It can be easily adapted to automate the installation of any guest VM using KVM on a Bare Metal instance.